static/js/modules/bar-charts.js

```diff
@@ -106,6 +109,7 @@ GroupedBarChart.prototype.buildChart = function() {
     .enter()
       .append('g')
       .attr('class', function(d) { return 'period ' + d.status; })
+      .attr('aria-labelledby', 'bar-chart-tooltip-' + this.index)
```

Comment:
@nickbristow : Is this the right place to add `aria-labelledby`? Hovering on the `g` element is what triggers the tooltip.